.github/workflows/build.yml

```diff
@@ -355,6 +356,13 @@ jobs:
         with:
           name: PrismLauncher-${{ matrix.name }}-${{ env.VERSION }}-${{ inputs.build_type }}
           path: PrismLauncher.tar.gz
+      
+      - name: Upload binary zip (macOS)
+        if: runner.os == 'macOS'
+        uses: actions/upload-artifact@v3
+        with:
+          name: PrismLauncher-${{ matrix.name }}-${{ env.VERSION }}-${{ inputs.build_type }}-zip
+          path: PrismLauncher.zip
```

Comment:
Wait... wouldn't this make a zip of a zip? Also you should update the release workflow